### PR TITLE
Drtii 823 log

### PR DESCRIPTION
**Pull request by ngu04**
- Remove the unused api for carrier code 
- using reference transparent logs 
- using joda datetime for timezone instead of Java ZoneDateTime
- adding NoPaxInfo for flight with no or zero  passenger information in totalpassenger of arrival table

### CODE REVIEW

File: src/main/scala/uk/gov/homeoffice/drt/repository/DepartureRepository.scala [hunk 1]
```diff
@@ -78,7 +78,7 @@ class DepartureRepository[F[_] : Sync](val sessionPool: Resource[F, Session[F]])
     val a: F[List[DepartureTableData]] = selectDepartureTableData(arrivalTableData)
     val insertDepartureData: F[Option[DepartureTableData]] = a.map { b =>
```
Comment (richbirch) [inline src/main/scala/uk/gov/homeoffice/drt/repository/DepartureRepository.scala:79]: What is `insertDepartureData`? It sounds like something that needs to be inserted, but it's begin returned by a function called `ignoreScheduledDepartureIfExist`. It's not clear what this function is doing given it's name

File: src/main/scala/uk/gov/homeoffice/drt/service/CiriumService.scala [hunk 2]
```diff
@@ -27,7 +28,7 @@ class CiriumService[F[_] : Sync](airlineConfig: AirlineConfig, client: Client[F]
           if (r.status == Status.Ok || r.status == Status.Conflict) {
```
Comment (richbirch) [inline src/main/scala/uk/gov/homeoffice/drt/service/CiriumService.scala:28]: Could we rename `process` to something more descriptive. I presume it's fetching missing departure times from the arrivals it's passed. If that's what it's doing then maybe call it something like `populateMissingDepartureTimes` or `fetchMissingDepartureTimes` if it's not actually updating anything (it looks like it returns the cirrium response)

File: src/main/scala/uk/gov/homeoffice/drt/service/FlightScheduledService.scala [hunk 3]
```diff
@@ -23,35 +24,37 @@ class FlightScheduledService[F[_] : Sync](arrivalsRepository: ArrivalRepositoryI
   def transformArrivals(requestedDetails: FlightsRequest, arrivalsTableData: F[List[ArrivalTableDataIndex]]): F[List[Arrival]] = {
```
Comment (richbirch) [inline src/main/scala/uk/gov/homeoffice/drt/service/FlightScheduledService.scala]: This is another function name that doesn't say what it does. How is it transforming arrivals? Is it taking rows from the arrivals table and turning them into `Arrival`s?

**Review by richbirch — APPROVED** [review]
:rocket:

File: src/main/scala/uk/gov/homeoffice/drt/repository/DepartureRepository.scala (hunk 1 above)
Comment (ngu04) [reply src/main/scala/uk/gov/homeoffice/drt/repository/DepartureRepository.scala:79]: `insertDepartureData` is just simple insert of Departure Data and it is part of DepartureRepository class . May I should rename it to `ignoreScheduledDepartureIfExist` to `upsertScheduledDeparture`

File: src/main/scala/uk/gov/homeoffice/drt/service/CiriumService.scala (hunk 2 above)
Comment (ngu04) [reply src/main/scala/uk/gov/homeoffice/drt/service/CiriumService.scala:28]: completely agree

File: src/main/scala/uk/gov/homeoffice/drt/service/FlightScheduledService.scala (hunk 3 above)
Comment (ngu04) [reply src/main/scala/uk/gov/homeoffice/drt/service/FlightScheduledService.scala]: I can rename as `transformArrivalsFromArrivalTable` which make is it more obvious .

**Pull request merged**